config/routes.rb

```diff
@@ -28,5 +28,6 @@
   scope module: :system_admin, path: "system-admin" do
     resources :applicants, only: %i[index show edit update]
     get "/dashboard", to: "dashboard#show"
+    resources "reports"
```

Comment:
we've only using index and show actions, we should restrict this with the :only keyword.
```suggestion
    resources "reports", only: [:show, :index]
```